changelog/betterc_cmp_types.dd

```diff
@@ -0,0 +1,42 @@
+Support for comparing more array types has been added to -betterC
```

Comment:
It would be nice to define "more". In your example, it's essentially character arrays (with support for switch) and static array ?